include/hip/devicelib/single_precision/sp_intrinsics.hh

```diff
@@ -24,6 +24,17 @@
 #define HIP_INCLUDE_DEVICELIB_SP_INTRINSICS_H
 
 #include <hip/devicelib/macros.hh>
+#include <hip/devicelib/single_precision/sp_math.hh>
```

Comment:
Why do we need this?

Reply:
The follow up implementations depends on its declarations.